docs/data/joy/components/aspect-ratio/aspect-ratio.md

```diff
@@ -3,22 +3,23 @@ product: joy-ui
 title: React Aspect Ratio component
 ---
 
-# Aspect ratio
+# Aspect Ratio
 
-<p class="description">The aspect ratio component shapes the content with the specified ratio.</p>
+<p class="description">The Aspect Ratio component resizes its contents to match the desired ratio.</p>
 
 ## Introduction
 
-`AspectRatio` is a wrapper component that allows you to rapidly control its content aspect ratio.
-Its default implementation combines `height: 0px` with percentage `padding-bottom` to properly accommodate the content.
+Aspect Ratio is a wrapper component for quickly resizing content to conform to your preferred ratio of width to height.
+Media content like images can be stretched, resized, and cropped based on the CSS `object-fit` property.
 
 :::info
-**Note:** A [native CSS `aspect-ratio` property](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) already exists but MUI is not using it yet due to limited browser support.
-Once the support increases significantly, MUI will switch over to it.
+A [native CSS `aspect-ratio` property](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) does exist, but MUI does not plan to implement it until browser compatibility increases.
```

Comment:
Is it really about waiting until compatibility increases? Because when I checked, it seems that pretty much all browsers support it now. Or is it that we need to support older browsers?

Reply:
The current browser support is at [~90%](https://caniuse.com/?search=aspect-ratio). Waiting for Safari to gain more adoption. I think around 94% is nice.

Reply:
OK cool, I think we should say that directly so that everyone is on the same page about what we mean when we say we're waiting for increased support.

Reply:
> we're waiting for increased support

That sounds better.

Reply:
I updated this text to be more specific:

> A [native CSS `aspect-ratio` property](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) does exist, but MUI does not plan to implement it until browser compatibility increases to at least 94%. As of Q4 2022, compatibility is at 90%. Source: [Can I use…](https://caniuse.com/?search=aspect-ratio)